test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -13123,6 +13123,15 @@ def is_cray(self):
         else:
             return True
 
+    def is_shasta(self):
+        """
+        Returns True if the version of PBS used is installed on Shasta platform
+        """
+        if self.platform == 'shasta':
+            return True
+        else:
+            return False
```

Comment:
This could all be replaced with
`return self.platform == 'shasta'`
But it's up to you if you want to change it.

Reply:
I like the readability of the code as it is.  :)